includes/Admin/AdminBar.php

```diff
@@ -108,19 +140,22 @@ public function visit_dashboard_menu( $wp_admin_bar ) {
             return;
         }
 
-        $dashboard = dokan_get_navigation_url();
-
-        if ( ! $dashboard ) {
-            return;
-        }
-
-        $wp_admin_bar->add_node(
-            [
-                'parent' => 'site-name',
-                'id'     => 'view-dashboard',
-                'title'  => __( 'Visit Vendor Dashboard', 'dokan-lite' ),
-                'href'   => $dashboard,
-            ]
-        );
+        $menus = $this->get_dokan_admin_menus();
+
+        // Show wp admin menus in admin bar here, if menu URL.
+        foreach ( $menus as $menu ) :
```

Comment:
Might consider not using conditional tags: https://developer.wordpress.org/themes/basics/conditional-tags/